src/ch/ch_conf.c

```diff
@@ -191,11 +247,7 @@ chExtractVersionInfo(int *retversion)
 
     tmp = help;
 
-    /* expected format: cloud-hypervisor v<major>.<minor>.<micro> */
-    if ((tmp = STRSKIP(tmp, "cloud-hypervisor v")) == NULL)
-        goto cleanup;
-
-    if (virParseVersionString(tmp, &version, true) < 0)
+    if (virCHVersionString(tmp, &version, true) < 0)
```

Comment:
`ret` is not set to return value from `virCHVersionString` here. Please fix.

Reply:
@praveen-pk  I am returning -1 in case of failed case and 0 in case of successful scenarion.